packages/js/e2e-environment/src/setup/jest.setup.js

```diff
@@ -175,6 +176,7 @@ beforeAll( async () => {
 	capturePageEventsForTearDown();
 	enablePageDialogAccept();
 	observeConsoleLogging();
+	setupJestRetries( 2 );
```

Comment:
We want this to be just the default parameter so when package users use `E2E_RETRY_TIMES` they don't need to add the function call to their local `jest.setup.js` for it to be applied.

Reply:
Good catch! I've removed the parameter from both instances where the `setupJestRetries` method is called.

Reply:
> I've removed the parameter from both instances where the setupJestRetries method is called.

@MrJnrman Sorry I wasn't clear. The reason we have the call in the package is to support the constant in another project if the constant is set. In the woocommerce repo we want retries enabled by default. 

The package (`packages/js/e2e-environment`) should be `setupJestRetries()`. The repo (`tests/e2e`) should be `setupJestRetries( 2 )`. Can you make a follow up PR to enable retries locally?